src/cmake/dependencies.cmake

```
@@ -10,6 +10,10 @@ if(NOT USE_SYSTEM_MAGNUM)
   # These are enabled by default but we don't need them right now -- disabling
   # for slightly faster builds. If you need any of these, simply delete a line.
   set(WITH_INTERCONNECT OFF CACHE BOOL "" FORCE)
+  # Ensure Corrade should be built statically if Magnum is.
+  set(BUILD_PLUGINS_STATIC ON CACHE BOOL "BUILD_PLUGINS_STATIC" FORCE)
+  set(BUILD_STATIC ON CACHE BOOL "BUILD_STATIC" FORCE)
+  set(BUILD_STATIC_PIC ON CACHE BOOL "BUILD_STATIC_PIC" FORCE)
```

Hm, can you then remove the same lines that are on line 175 below? Ideally also put a comment there that a static build is being enabled earlier.

It can also be without the docstring, it's useless anyway:

```suggestion
  set(BUILD_PLUGINS_STATIC ON CACHE BOOL "" FORCE)
  set(BUILD_STATIC ON CACHE BOOL "" FORCE)
  set(BUILD_STATIC_PIC ON CACHE BOOL "" FORCE)
```